templates/toil-infra-essentials.yaml

```diff
@@ -22,6 +22,7 @@ Resources:
                     - ':root'
                 - !ImportValue us-east-1-bootstrap-TravisUserArn
                 - !ImportValue us-east-1-bootstrap-CfServiceRoleArn
+                - "arn:aws:iam::563295687221:role/accounts-AWSIAMAdminRole-1B5HUQPC19H69"
```

Comment:
this is ONLY for sandbox, it's different for scicomp.  You'll need to add [conditional logic](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/conditions-section-structure.html) to set this for sandbox or scicomp depending on which ENV you run it for.  Condition is used in  toil-instance.yaml

Reply:
the equivalent role in scicomp is "arn:aws:iam::055273631518:role/accounts-AWSIAMAdminRole-JL0BF315YDU0"